### ISSUE TITLE
Travis CI build fails

### ISSUE THREAD

**Issue by sriemer** [scanmem/scanmem#319]
There is a gcc-6 bug in the Ubuntu toolchain PPA:
https://stackoverflow.com/questions/50024731/ld-unrecognized-option-push-state-no-as-needed

**Comment by 12345ieee** [comment]
So exactly our setup (recent gcc with UB sanitizer) is broken, what a joy.

I've seen you try some variations under the `travis` branch, did you have any success?

**Comment by sriemer** [comment]
Nope, no success so far. A Bionic build worker should fix this but Bionic is not available. No idea how to change the linker with autotools.
